.github/actions/slack-on-error/action.yaml

```diff
@@ -0,0 +1,81 @@
+# This is a GitHub Action to report an error to Teleport internal slack.
+
+name: Slack on Error
+description: Report CI errors to a slack channel
+
+inputs:
+  slack-token:
+    description: Slack bot token.
+    required: true
+  channel-id:
+    description: ID of channel to send error to.
+    default: 'C052S7U6SR1' # #drone-alerts channel. Override for desired channel
```

Comment:
I do not want alerts in #internal-tools. Errors from builds are not necessarily relevant to internal tools. I expect we'll rename #drone-alerts to #ci-alerts, but that should not change the channel ID (I don't really know, but it would defeat the purpose of an ID for it to change if the name changes).